sections/main-cart-items.liquid

```diff
@@ -105,6 +105,28 @@
                         </li>
                       {%- endfor -%}
                     </ul>
+                    {%- if item.original_price != item.final_price -%}
+                      <dl class="cart-item__discounted-prices">
+                        <dt class="visually-hidden">
+                          {{ 'products.product.price.regular_price' | t }}
+                        </dt>
+                        <dd>
+                          <s class="cart-item__old-price product-option">
+                            {{ item.original_price | money }}
+                          </s>
+                        </dd>
+                        <dt class="visually-hidden">
+                          {{ 'products.product.price.sale_price' | t }}
+                        </dt>
+                        <dd class="product-option">
+                          {{ item.final_price | money }}
+                        </dd>
+                      </dl>
+                    {%- else -%}
+                      <span class="product-option">
+                        {{ item.original_price | money }}
+                      </span>
+                    {%- endif -%}
```

Comment:
Shouldn't this be before the product options/variants ?
![](https://screenshot.click/21-10-4x8uc-xbral.png)
vs
![](https://screenshot.click/21-10-tamdi-pkgsb.png)